examples/geometry/juelich_tests/test12.wkt

```diff
@@ -0,0 +1 @@
+GEOMETRYCOLLECTION (POLYGON ((5.42159 4.73707, 8.42159 4.73707, 8.42159 4.83707, 5.42159 4.83707, 5.42159 6.78707, 8.42159 6.78707, 11.4216 6.78707, 11.4216 2.78707, 8.42159 2.78707, 5.42159 2.78707, 5.42159 4.73707), (8.64584 5.02285, 8.64584 4.78707, 11.3378 4.78707, 11.3378 5.02285, 8.64584 5.02285)))
```

Comment:
<img width="989" alt="Screenshot 2023-04-21 at 15 38 50" src="https://user-images.githubusercontent.com/13482142/233650409-755b07ba-9537-4557-802c-9af401c5c2f7.png">

Both gaps here are too narrow and not supported right now. Can you please ask @chraibi / @JetteSchumann what the intent was with this geometry?

Maybe we can change the geometry and still can test as intended. If we cannot figure out what the idea was just drop this file.

Reply:
What's with this one? Did you @ChristianHirt talk to @JetteSchumann or @chraibi ?

Reply:
It seems that this file was used to test if the pedestrian can see the door even if there is an obstacle. We decided that this geometry has no further use for us and can be removed.